### PR TITLE
Emp tweaks

### PR DESCRIPTION
**Pull request by hex37**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:
Increases internal damage from EMPs on robotic prosthetics
Severely reduces external damage from EMPs on Robotic Prosthetics
Reduces internal damage from EMPs on prosthetics contained within biological organs
/:cl:

### CODE REVIEW

**Comment by r4d6** [comment]
Just so that you know, _**NOTHING**_ deals an EMP of severity 2 or 3. Ever.

File: code/modules/organs/external/_external.dm [hunk 1]
```diff
@@ -279,14 +279,18 @@
 /obj/item/organ/external/emp_act(severity)
 	if(!BP_IS_ROBOTIC(src))
 		return
+	
+	if(severity == 1)
```
Comment (Trilbyspaceclone) [inline code/modules/organs/external/_external.dm:283]: 5 out of **60** for being hit with the heavest EMP is a bit low

Comment (Trilbyspaceclone) [reply code/modules/organs/external/_external.dm:283]: maybe have it be like 5/3/1?

**Comment by Trilbyspaceclone** [comment]
Also emps do use the servarity system

**Comment by r4d6** [comment]
> Also emps do use the servarity system

Yes, but they all deal Severity 1 by default.
I don't think I ever saw something deal Severity 2 or 3.

**Comment by Trilbyspaceclone** [comment]
Stuff does infact use and proc 2 and 3

**Comment by Trilbyspaceclone** [comment]
I would tbh flip this were it deals tons of damage to lims minor damage to orangs personally but please do a 5/3/1 for it

**Comment by hex37** [comment]
@Trilbyspaceclone 

Deal. I fixed the external value to be 5/3/1

I do understand that having it deal external damage is more funny, but I decided against it for two reasons.

1. EMPs don't shred metal, they fry internal circuitry and PSU's. Wires and regular metal is largely unaffected by them, but internal parts should be _ruined_ by it.
2. EMPs were headgibbing synths. I'm totally fine with them killing robots, nuts to them OP robots, but I don't want them to be round-removed for getting a heavy EMP to the face. Now they take more damage that they need to sit down and have fixed for them. 

Essentially, I see this as an synth nerf made out of pure consideration for Synths.

**Review by Trilbyspaceclone — APPROVED** [review]
Looks good to me

**Pull request merged**